### ISSUE TITLE
Bug - sev-2 - ENG: Update buttons and link in Rx

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7807]
The following list are the updates that need to be made to buttons and links in Rx within the app.
_This list has been pulled in from the [full list of audit results](https://docs.google.com/spreadsheets/d/1uOF4dn3yptsp7VnxwG7owy2p0GDHjmBq1oBR8MIVvWg/edit?usp=sharing)._
- [ ] **[prescriptions status]**
     - **Location:** Health / Prescriptions, Health / Prescriptions / Details
     - **Current component type:** Button (says both link and button)
     - **Current style:** N/A
     - **Icon:** Information
     - **What happens?** Launches the status definition full panel
     - **Recommended change:** Should be a link
- [ ] **Get prescription details**
     - **Location:** Health / Prescriptions
     - **Current component type:** Button
     - **Current style:** N/A
     - **Icon:** Chevron Right
     - **What happens?** Redirects you to the prescription details child screen
     - **Recommended change:** Should be a link not a button
- [ ] **Get prescription tracking**
     - **Location:** Health / Prescriptions
     - **Current component type:** Button
     - **Current style:**  N/A
     - **Icon:** Truck
     - **What happens?** Launches full panel with prescription tracking information
     - **Recommended change:** Should be a link not a button

**Comment by brea11y** [comment]
Ticket has been reviewed and is still valid.

**Comment by DJUltraTom** [comment]
verified changes listed in the AC are being presented as expected.
QA approves this change

**Comment by DJUltraTom** [comment]
verified this is functioning in the overnight build, 
closing as deployed
